domains/sreerag.json

```diff
@@ -1,9 +1,14 @@
 {
   "owner": {
+    "github": "sreeraganil",
     "username": "sreeraganil",
     "email": "sreeraganil2003@gmail.com"
   },
   "records": {
-    "URL": "https://sreerag.deno.dev"
-  }
+    "CNAME": "sreerag.deno.dev",
+    "TXT": {
+      "_acme-challenge": "8b038e69da7b0f969ffcf04c._acme.deno.dev"
+    }
+  },
+  "proxied": true
 }
```

Comment:
File does not follow proper schema. See docs for more information: https://docs.is-a.dev/